VAMobile/src/components/NavigationTabBar.test.tsx

```diff
@@ -26,86 +17,54 @@ context('NavigationTabBar', () => {
     { name: 'Payments', key: 'Payments-1' },
   ]
 
-  const initializeTestInstance = (index = 0, routesList = routes) => {
-    emitSpy = jest.fn(() => {})
-    navigateSpy = jest.fn(() => {})
-
-    component = render(
+  const renderWithRoute = (index = 0) => {
+    jest.resetAllMocks()
+    render(
       <NavigationTabBar
-        state={{ index, routes: routesList } as unknown as TabNavigationState<ParamListBase>}
+        state={{ index, routes } as unknown as TabNavigationState<ParamListBase>}
         navigation={{ emit: emitSpy, navigate: navigateSpy } as unknown as NavigationHelpers<ParamListBase, BottomTabNavigationEventMap>}
-        translation={t}
+        translation={jest.fn()}
       />,
     )
-
-    testInstance = component.UNSAFE_root
   }
 
   beforeEach(() => {
-    initializeTestInstance()
+    renderWithRoute(0)
   })
 
-  it('initializes correctly', async () => {
-    expect(component).toBeTruthy()
+  it('navigates when tab is pressed', () => {
+    fireEvent.press(screen.getByRole('tab', { name: 'Home' }))
+    expect(emitSpy).toBeCalled()
   })
 
-  describe('when a tab option is pressed', () => {
-    it('should call the navigation emit spy', async () => {
-      await waitFor(() => {
-        testInstance.findAllByType(TouchableWithoutFeedback)[0].props.onPress()
-        expect(emitSpy).toBeCalled()
-      })
-    })
-
-    describe('when isFocused is false and navigation emit returns false for defaultPrevented', () => {
-      it('should call navigation emit and navigate spy', async () => {
-        await waitFor(() => {
-          emitSpy.mockReturnValue({ defaultPrevented: false })
-          testInstance.findAllByType(TouchableWithoutFeedback)[1].props.onPress()
-          expect(emitSpy).toBeCalled()
-          expect(navigateSpy).toBeCalled()
-        })
-      })
-    })
+  it('navigates when tab is long pressed', async () => {
+    await userEvent.longPress(screen.getByRole('tab', { name: 'Benefits' }))
```

Comment:
Had to upgrade @testing-library/react-native to get `userEvent.longPress`. Tests seem to run fine with the newer version